fetcher-pipeline/Dockerfile

```diff
@@ -17,4 +17,6 @@ RUN curl -O https://download.java.net/java/GA/jdk11/9/GPL/openjdk-11.0.2_linux-x
     tar zxvf openjdk-11.0.2_linux-x64_bin.tar.gz && mv jdk-11* /usr/local/
 
 COPY entrypoint.sh /entrypoint.sh
+COPY gl_repo_urls.txt $ORATURE_REPO_DIR
```

Comment:
I think $ORATURE_REPO_DIR should be replaced with an absolute path in the container that your script references by absolute path inside the container.

Reply:
the python script also uses the env var to work with repo directory

`        self.repo_dir = os.getenv("ORATURE_REPO_DIR")`

Reply:
Dockerfile is different. We're not passing env vars in at build time. env vars are for runtime. This is a statically loaded file at build time so an absolute path is advised. I'd recommend treating the repo list and the temp dir as separate.

Reply:
okay sure. I've made the change.

Reply:
looks good